kotlin-jvm/src/main/kotlin/playground/SkrapeIt.kt

```diff
@@ -179,8 +179,8 @@ private fun loadReferenceResult(): String {
         .readText()
 }
 
-private fun getMockHtml(): String {
-    return """
+private fun getMockHtml(): String =
+    """
```

Comment:
Why keep the line break here?

Reply:
I was not quite sure on the expected styling. Was doing my best to follow the code convention

> If the function has an expression body that doesn't fit in the same line as the declaration, put the = sign on the first line. Indent the expression body by four spaces.

https://kotlinlang.org/docs/reference/coding-conventions.html#expression-body-formatting

Reply:
This is when the first line of the body doesn't fit on the line, but it's not the case here.
I submitted an edit to improve it the convention wording and example: https://github.com/JetBrains/kotlin-web-site/pull/1925

Can you open a new PR to improve this here?